README.md

```diff
@@ -11,111 +16,148 @@
+- [Impact Client](https://impactclient.net/)
+- [Meteor Client](https://meteorclient.com/)
+- [Aristois](https://aristois.net/)
+  
+- Note: Many utility clients offer Baritone, but sometimes utility clients can come with **malware**,
+use your best judgement and be careful of what you download.
 
-[**Baritone Discord Server**](http://discord.gg/s6fRBAUpmr)
+Installing a utility client with Baritone support is an easy way to get started with Baritone
+as most utility clients include a GUI to navigate Baritone's commands.
 
-Baritone is the pathfinding system used in [Impact](https://impactclient.net/) since 4.4. There's a [showcase video](https://youtu.be/CZkLXWo4Fg4) made by @Adovin#0730 on Baritone which I recommend. [Here's](https://www.youtube.com/watch?v=StquF69-_wI) a (very old!) video I made showing off what it can do. [Tutorial playlist](https://www.youtube.com/playlist?list=PLnwnJ1qsS7CoQl9Si-RTluuzCo_4Oulpa)
+The safest way to install Baritone is to install a release .jar from this repository 
+and place it in your .minecraft\mods\ folder.
 
-The easiest way to install Baritone is to install [Impact](https://impactclient.net/), which comes with Baritone. The second easiest way (for 1.12.2 only) is to install the v1.2.* `api-forge` jar from [releases](https://github.com/cabaletta/baritone/releases). **For 1.12.2 Forge, just click [here](https://github.com/cabaletta/baritone/releases/download/v1.2.15/baritone-api-forge-1.2.15.jar)**. Otherwise, see [Installation & setup](SETUP.md). Once Baritone is installed, look [here](USAGE.md) for instructions on how to use it.
+The most recent version of Minecraft that Baritone currently supports is *1.16.5* ,
+Baritone will be updated to 1.17.~ soon.
 
-For 1.15.2, [click here](https://www.youtube.com/watch?v=j1qKtCZFURM) and see description. If you need Forge 1.15.2, look [here](https://github.com/cabaletta/baritone/releases/tag/v1.5.3), follow the instructions, and get the `api-forge` jar.
+## Getting Started
 
-For 1.16.5, [click here](https://www.youtube.com/watch?v=_4eVJ9Qz2J8) and see description. If you need Forge or Fabric 1.16.5, look [here](https://github.com/cabaletta/baritone/releases/tag/v1.6.3) and get the `api-forge` or `api-fabric` jar. **For 1.16.5 Fabric, just click [here](https://github.com/cabaletta/baritone/releases/download/v1.6.3/baritone-api-fabric-1.6.3.jar)**.
+- #### [Features](FEATURES.md)
 
-This project is an updated version of [MineBot](https://github.com/leijurv/MineBot/),
-the original version of the bot for Minecraft 1.8.9, rebuilt for 1.12.2 through 1.16.5. Baritone focuses on reliability and particularly performance (it's over [30x faster](https://github.com/cabaletta/baritone/pull/180#issuecomment-423822928) than MineBot at calculating paths).
+- #### [Installation & setup](SETUP.md)
 
-Have committed at least once a day from Aug 1, 2018, to Aug 1, 2019.
+- #### [API Javadocs](https://baritone.leijurv.com/)
```

Comment:
Maybe this would be a good place for a a link to a description on how the api works?
Not telling you to it, just asking.